scalafix-tests/input/src/main/scala/test/Disable.scala

```diff
@@ -52,4 +56,10 @@ If you must Option.get, wrap the code block with
 ...
 // scalafix:on Option.get
 */
-}
\ No newline at end of file
+  val l: ListBuffer[Int] =
+    scala.collection.mutable.ListBuffer.empty[Int] // scalafix:ok Disable.ListBuffer
```

Comment:
:thinking: why is it ok?

Reply:
`// scalafix:ok` is about the same as `// assert: `, since scalafix will report an error on `// scalafix:ok` if it's not used. Maybe we should consider removing `// scalafix: assert` 🤔 but assert also has the option to test the exact error message.